iSkyLIMS_drylab/views.py

```diff
@@ -139,19 +145,23 @@ def request_sequencing_service(request):
                 },
             )
 
-        new_service = create_new_save_sequencing_service_request(request)
-        sample_stored = stored_samples_for_sequencing_request_service(
+        new_service = iSkyLIMS_drylab.utils.handling_request_services.create_new_save_sequencing_service_request(
+            request
+        )
+        sample_stored = iSkyLIMS_drylab.utils.handling_request_services.stored_samples_for_sequencing_request_service(
             request.POST, new_service
         )
         if "files" in request.POST:
-            add_files_to_service(request.POST.getlist("files"), new_service)
+            iSkyLIMS_drylab.utils.handling_request_services.add_files_to_service(
+                request.POST.getlist("files"), new_service
+            )
         ## Send mail to user and drylab notification email
         email_data = {}
         if (
             "requestedForUserid" in request.POST
             and request.POST["requestedForUserid"] != ""
         ):
-            user_obj = User.objects.filter(
+            user_obj = iSkyLIMS_drylab.models.User.objects.filter(
```

Comment:
User class belongs to django framework , it does not part of  iSkyLIMS_drylab.models
keep only **User.objects.filter** remove the iSkyLIMS_drylab.models

I have realize that the import is missing, Could you please add the following line 
from django.contrib.auth.models import User